patch/0012-arm64-dts-marvell-Add-Supermicro-SSE-G3748-board.patch

```diff
@@ -0,0 +1,557 @@
+diff --git a/arch/arm64/boot/dts/marvell/Makefile b/arch/arm64/boot/dts/marvell/Makefile
```

Comment:
Please add patch files creates with `git format-patch` containing a suitable and elaborate commit message including Signed-off-by lines